src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -201,6 +248,21 @@ const TopicFragment = ({
           </View>
         </Pressable>
         <FlatList
+          ListHeaderComponent={() => (
+            <>
+              <RecentSearch
+                shown={!withoutRecent || isFirstTimeOpen}
+                setSearchText={setSearchText}
+                setIsFirstTimeOpen={setIsFirstTimeOpen}
+              />
+              <AccordionView
+                data={firstData}
+                renderItem={renderItem}
+                activeSections={activeSections}
+                setActiveSections={setActiveSections}
+              />
+            </>
+          )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ListHeaderComponent` prop of the `FlatList` component now includes the `RecentSearch` and `AccordionView` components. This is a good use of the `ListHeaderComponent` prop to render additional content above the list.
```diff
+ ListHeaderComponent={() => (
+   <>
+     <RecentSearch
+       shown={!withoutRecent || isFirstTimeOpen}
+       setSearchText={setSearchText}
+       setIsFirstTimeOpen={setIsFirstTimeOpen}
+     />
+     <AccordionView
+       data={firstData}
+       renderItem={renderItem}
+       activeSections={activeSections}
+       setActiveSections={setActiveSections}
+     />
+   </>
+ )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->